Update dashboard_block.html | To hide the "My Buddies" sub-block

BDadmehr0 opened the pull request:
To hide the "My Buddies" sub-block, you can simply remove the code block that renders it. Here's the modified code without the "My Buddies" sub-block:

BDadmehr0 committed:
Update dashboard_block.html

thejoeejoee commented:
@BDadmehr0 That's not the right solution how it should work. The block should stay here, the only needed thing is to display CTA (instead of matched buddies) for people without any matched buddies. The CTA should target the requests matching page.

BDadmehr0 committed:
Update dashboard_block.html

Based on your description, here's the modified code to display a call-to-action (CTA) for people without any matched buddies on the requests matching page

BDadmehr0 committed:
Merge branch 'develop' into develop
